### ISSUE TITLE
Does not work with Tetra weapons

### ISSUE THREAD

**Issue by K1teenz** [millennIumAMbiguity/no-more-knockback-latency#1]
Mod version: 0.0.3
MC version: 1.20.1
Forge version: 47.3.0

I was playing this decently large modpack that I setup with mods including this mod & Tetra by Mickelus, and it seems that the weapons/tools that you can make in Tetra/Tetra addons deal their attack speed as damage instead of their attack damage as damage when this mod is active at the same time. It also happened a few times with vanilla weapons as well, but it seems really inconsistent overall. It was also sometimes working normally, but most of the time it worked as I said before. Tetra is a pretty popular mod so I'd consider this a notable issue.

**Comment by millennIumAMbiguity** [comment]
Is this issue present only when this mod and Tetra are installed?

**Comment by K1teenz** [comment]
Yes, and I tested by using a Tetra modified sword that had a 7.5 attack damage (there werent any special modifiers causing the tool to deal less damage) stat, and without this mod, a Creeper (with 20 health) would die in 3 hits, but adding the mod resulted in more than 3 hits. I tried the same with a Zoglin (40 health iirc), without this mod would die in 6 hits, with this mod it would take more. Using a damage number mod and Jade also showed how much damage it was dealing, although I did test without any other mods as I said above.
